spirv_reflect.c

```diff
@@ -270,23 +290,24 @@ static bool SearchSortedUint32(const uint32_t* arr, size_t size, uint32_t target
   return false;
 }
 
-static SpvReflectResult IntersectSortedUint32(const uint32_t* p_arr0, size_t arr0_size, const uint32_t* p_arr1, size_t arr1_size,
-                                              uint32_t** pp_res, size_t* res_size) {
+static SpvReflectResult IntersectSortedAccessedVariable(const SpvReflectPrvAccessedVariable* p_arr0, size_t arr0_size,
```

Comment:
This should be one param per line.

Reply:
https://stackoverflow.com/questions/29483626/clang-format-force-arguments-and-parameters-to-one-line-each-if-the-statement-o

we would need to set `BinPackParameters` in the `.clang-format` file